src/pages/blog/starlingx-ptp-multi-instance-features.md

```diff
@@ -0,0 +1,126 @@
+---
+templateKey: blog-post
+title: Precision Time Protocol Enhancements StarlingX 7.0
+author: Cole Walker
+date: 2022-04-19

```

Comment:
```suggestion
date: 2022-06-05

```

Reply:
Scheduling this a few weeks out since we still have a 'part 2' of the PTG recap blog post coming out.